Documentation/microsoft-git.md

```diff
@@ -0,0 +1,39 @@
+Details about the Microsoft/Git Fork
+====================================
+
+The [`microsoft/git` fork of Git](https://github.com/microsoft/git)
+is used to take early performance patches or custom patches for
+[VFS for Git](https://github.com/microsoft/git).
```

Comment:
Should the URL here bit microsoft/git?  or should it be VFS?
Or should this be "... early performance or custom patches in Git for use in VFS4G..." ?